### PR TITLE
Update Italian localization

### PR DESCRIPTION
**Pull request by M0Rf30**
As in title,
Regards

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -2243,39 +2258,44 @@ msgstr "Copia _URL"
...
+#: pynicotine/gtkgui/search.py:576
 msgid "Filtered out inexact or incorrect search result "
 msgstr "Risultati di ricerca inesatti o incorretti esclusi dal filtro "
 
-#: pynicotine/gtkgui/search.py:925 pynicotine/gtkgui/search.py:927
-msgid "Result Filters"
-msgstr "Filtri Risultato"
+#: pynicotine/gtkgui/search.py:926
+#, python-format
+msgid "Result _Filters [%d]"
```
Comment (slook) [inline po/it.po:2291]: Key accelerator shortcut will be changed to "_Result Filters" see https://github.com/nicotine-plus/nicotine-plus/issues/1500#issuecomment-911158480 sorry. s.

File: po/it.po [hunk 2]
```diff
@@ -5739,27 +5707,27 @@ msgstr "Dimensione file"
...
+#: pynicotine/gtkgui/ui/search.ui:309
 #: pynicotine/gtkgui/ui/settings/search.ui:349
 msgid "Country code, e.g. US|GB|ES or !DE|!GB"
 msgstr "Codice paese, es. US|GB|ES o !DE|!GB"
 
 # pynicotine/gtkgui/nicotine_glade.py
 # pynicotine/gtkgui/settings_glade.py
-#: pynicotine/gtkgui/ui/search.ui:322
+#: pynicotine/gtkgui/ui/search.ui:326
 msgid "Free _Slot"
```
Comment (slook) [inline po/it.po:5722]: Key accelerator shortcut will be changed to "Free Slo_t" see #1516

**Review by slook — CHANGES_REQUESTED** [review]
- [ ] "Result _Filters" -> "_Result Filters" - see https://github.com/nicotine-plus/nicotine-plus/issues/1500#issuecomment-911158480 - THIS ISSUE NEEDS TO BE RE-OPENED (sorry my mistake) s.

- [ ] "Free _Slot" -> "Free Sl_ot" - see #1516 
 
- [ ] "Send _Message" -> "Send M_essage" - see #1515

- [ ] "Brow_se Files" -> "_Browse Files"

**Comment by mathiascode** [comment]
@M0Rf30 Thank you. I haven't merged this yet due to several strings being refined in master at the moment, but this should hopefully be done soon.

**Pull request merged**